app/src/main/java/co/nimblehq/alpine/sample/ui/camera/CameraCaptureActivity.kt

```diff
@@ -30,9 +36,7 @@ class CameraCaptureActivity : ComponentActivity() {
     private val loadingDialog by lazy {
         createLoadingDialog()
     }
-    private val mrzProcessor: MrzProcessor by lazy {
-        MrzProcessor.newInstance()
-    }
+    private val viewModel: CameraCaptureViewModel by provideViewModels()
```

Comment:
Why don't we just use `by viewModels()` instead? 🤔 
```suggestion
    private val viewModel: CameraCaptureViewModel by viewModels()
```

Reply:
Oh, yeah I think It better let me update 👍